### PR TITLE
[DO-416] Restructure Gridsome plugins

### PR DESCRIPTION
**Pull request by d0rich**

### CODE REVIEW

File: gridsome.config.js [hunk 1]
```diff
@@ -85,19 +85,11 @@ module.exports = {
...
     },
-      // Terms Nodes
-    {
-      use: '@gridsome/source-filesystem',
-      options: {
-        path: './content/terms/**/*.md',
-        typeName: 'Term'
-      }
-    },
-    require('./src/plugins/gridsome-plugin-th2-normalize-docs'),
```
Comment (StanG4313) [inline gridsome.config.js:96]: Why normalize-docs were renamed to terms?

File: src/plugins/remark/default-code-snippet-lang/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Default code snippet language
+
+Code snippets without defined programming language displays some symbols incorrectly, e.g.: `<`, `>`, `{`, `}`.
+
+So this plugin will define programming language for code snippets without specified language. 
+
+Default language: `shell`
```
Comment (StanG4313) [inline src/plugins/remark/default-code-snippet-lang/README.md:7]: Should we add info about how to choose non-default language, or it is too obvious?

File: gridsome.config.js (hunk 1 above)
Comment (d0rich) [reply gridsome.config.js:96]: This plugin was created initially to manage versioned folders. Now it only connect pages and terms.

File: src/plugins/remark/default-code-snippet-lang/README.md (hunk 2 above)
Comment (d0rich) [reply src/plugins/remark/default-code-snippet-lang/README.md:7]: Non-default is any language specified. It is part of markdown syntax.

Comment (d0rich) [reply src/plugins/remark/default-code-snippet-lang/README.md:7]: It is very hard (or impossible) to add markdown code snippet inside markdown document.

File: src/plugins/gridsome/plugin/th2-check-content/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Check content plugin
+
+Plugin checks content for forbidden words or collocations and drop build process if some exists.
```
Comment (kapuTheblack) [inline src/plugins/gridsome/plugin/th2-check-content/README.md:3]:
```suggestion
Plugin checks content for forbidden words or collocations and drops build process if some exists.
```

File: src/plugins/gridsome/plugin/th2-check-content/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+## Requirements
+
+Environment variables with words or collocations listed through ",".
```
Comment (kapuTheblack) [inline src/plugins/gridsome/plugin/th2-check-content/README.md:7]:
```suggestion
Environment variables with words or collocations are listed through ",".
```

**Review by kapuTheblack — COMMENTED** [review]
Some additional minor suggestions.

**Review by StanG4313 — APPROVED** [review]

**Pull request merged**